test/unit/jest.config.js

```diff
@@ -4,9 +4,9 @@
 const glob = require( 'glob' ).sync;
 
 // Finds all packages which are transpiled with Babel to force Jest to use their source code.
-const transpiledPackageNames = glob( 'packages/*/src/index.js' ).map(
-	( fileName ) => fileName.split( '/' )[ 1 ]
-);
+const transpiledPackageNames = glob(
+	'packages/*/src/index.{jsx,js,ts,tsx}'
```

Comment:
We don't use `jsx` but I guess for TS you need to keep `tsx` for some reasons.

Reply:
Yeah for TypeScript JSX syntax is only allowed in `tsx` files. I'll remove `jsx` from this list, I didn't realize we don't have any of it at all.